server/config.ts

```diff
@@ -70,8 +70,8 @@ export default {
     licenceApi: {
       url: get('LICENCE_API_URL', 'http://localhost:8089', requiredInProduction),
       timeout: {
-        response: get('LICENCE_API_TIMEOUT_RESPONSE', 10000),
-        deadline: get('LICENCE_API_TIMEOUT_DEADLINE', 10000),
+        response: get('LICENCE_API_TIMEOUT_RESPONSE', 30000),
```

Comment:
Is this because the image endpoint can be slow?

Reply:
Yes - on the first call to pdfDoc it needs to load fonts and with a 10 sec timeout this did time-out once when running locally. However, subsequent calls are very quick. I only saw this once so not sure that 30s is the right value.